packages/embed/e2e/cypress-utils.js

```diff
@@ -0,0 +1,31 @@
+export const IFRAME_SELECTOR = '[data-qa="iframe"]'
+export const screenSizeDesktop = { width: 1024, height: 768 }
+export const screenSizeMobile = { width: 375, height: 667 }
+
+const setViewport = ({ width, height }) => {
+  cy.viewport(width, height)
+}
+
+export const getIframeBody = (iframe = IFRAME_SELECTOR) => {
```

Comment:
Is this used anywhere? I would like to avoid this, we should not test <iframe> body anyway, but rather its attributes (eg. `src`). What is inside an iframe is displayed by renderer.

Also, typescript for this file too.